content/backend/graphql-php/4-summary.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Summary
+pageTitle: "Building a GraphQL Server with PHP using Laravel Framework"
+description: "You learned how to build a GraphQL server with PHP using Laravel Framework and Lighthouse package"
+---
+
+We hope this example shows a taste of the power of GraphQL and how Lighthouse makes it easy to build your own server with Laravel. 
+
+The app you just build might use some more features. Here are a few ideas on what you might add
+to learn more about Lighthouse.
+
+- Add pagination to your fields
```

Comment:
Maybe add links to where you can find the information on how to do this in the Lighthouse docs?